submit/sample.sh

```diff
@@ -0,0 +1,32 @@
+## Successful
+input="gs://cpg-tob-wgs-test/mt/v7.mt"
+outputDir="mt-call-rate-test"
+chr=22
+analysis-runner --dataset sgs-somatic-mtn --access-level test  --output-dir "$outputDir" --description "pVCF" sub_dataproc.py \
+    --script "scripts/mt_vcf.py --dataset '$input' --chrom chr${chr} --output Check_chr${chr}.vcf.bgz" \
```

Comment:
It looks like these scripts are no longer in the directory - is this sample.sh script actually being used, or has it been replaced by the analysis-runner commands in Submit_Jobs? If it's not being used anymore it would be great if you could remove it to avoid confusion.

Reply:
Done!